### PR TITLE
Fix all combinations

### PR DESCRIPTION
**Pull request by mayayat**

### CODE REVIEW

File: src/testGenAllCombinations/TestGenAllCombinations.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		ExpectedList.add(meet3);
+		ExpectedList.add(meet4);
+		boolean result = false;
+		
+		MeetingSetinACourse = c1.generateMeetingSet(m1, m2);
+		for( int i=0; i<MeetingSetinACourse.size(); i++){
+			if(MeetingSetinACourse.get(i).equals(ExpectedList.get(i))){
+			result = true;
+			}
+		}
```
Comment (vicksonzero) [inline src/testGenAllCombinations/TestGenAllCombinations.java:47]: to check if all of them equals, you should 
1. set result = true at first,
2. for all meetingsets created:
3. if they are not equal, set result = false

in your code here, if any 1 pair of meeting sets are equal, the test will pass

File: src/timetableGen/Timetable.java [hunk 2]
```diff
@@ -53,4 +55,16 @@ public boolean hasConflict(){
 		return false;
 	}
 	
-}
+	public void genAllCombinations(Course course1, Course course2){
```
Comment (vicksonzero) [inline src/timetableGen/Timetable.java:58]: all combinations should pick 1 meeting set from every course, not just 2 courses. you may try recursion in making this kind of situation. if you really want to use a loop, please make use of a stack structure

**Comment by vicksonzero** [comment]
please respond to the above comments (either fix it or discuss it) and merge master before reopening this merge request

**Comment by mayayat** [comment]
@vicksonzero can you explain more details about how to implement recursion in this method?

**Pull request closed without merging**